04-onion-routing.md

```diff
@@ -780,25 +793,26 @@ The _origin node_:
     - otherwise:
       - if UPDATE is set, AND the `channel_update` is valid and more recent
       than the `channel_update` used to send the payment:
-        - if this [FIXME: what does 'this' refer to?] should not have caused the failure:
+        - if this [FIXME: what does 'this' refer to?] should NOT have caused the failure:
```

Comment:
the `channel_update` included in the error packet.

Reply:
fixed